test/orb.jl

```diff
@@ -33,7 +33,7 @@ facts("ORB") do
         desc_2, ret_keypoints_2 = create_descriptor(img_array_2, orb_params)
         matches = match_keypoints(ret_keypoints_1, ret_keypoints_2, desc_1, desc_2, 0.2)
         reverse_keypoints_1 = [_reverserotate(m[1], pi / 4, (256, 384)) for m in matches]
-        @fact all(isapprox(rk[1], m[2][1], atol = 3) && isapprox(rk[2], m[2][2], atol = 3) for (rk, m) in zip(reverse_keypoints_1, matches)) --> true
```

Comment:
@annimesh2809, @timholy how about adding a function like `grade_matches` or something which takes in two sets of keypoints and returns a score for the quality of matches? It will be useful for object tracking also.